client/src/components/RegisterPage/Registration.js

```diff
@@ -100,10 +119,14 @@ const Register = (props) => {
   // Make sure to place proper link going into the login page in the href
   return (
     <div>
-      <h1 className="Title">Register</h1>
+      <h1 className="Title">
+        <StyledText>Register</StyledText>
+      </h1>
       <div className="box">
```

Comment:
See comment above about adding ``` text-align: center; ``` to StyledText rather than wrap it in ```<center> and <h1>```

Reply:
fixed